### PR TITLE
cocoapods-dependency-check 0.1

### PR DESCRIPTION
**Pull request by johnjcsmith**
### New Pull Request Checklist

*Please mark the points which you did / accept.*

- [x ] __I will not move an already shared step version's tag to another commit__
- [ x] I read the [Step Development Guideline](https://github.com/bitrise-io/bitrise/blob/master/_docs/step-development-guideline.md)
- [ x] I have a test for my Step, which can be run with `bitrise run test` (in the step's repository)
- [ x] I did run `bitrise run audit-this-step` (in the step's repository - note, if you don't have this workflow in your `bitrise.yml`, [you can copy it from the step template](https://github.com/bitrise-steplib/step-template/blob/master/bitrise.yml).)
- [ x] I read and accept the [Abandoned Step policy](https://github.com/bitrise-io/bitrise-steplib#abandoned-step-policy)

### CODE REVIEW

File: steps/cocoapods-dependency-check/0.1/step.yml [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+website: https://github.com/johnjcsmith/bitrise-step-cocoapods-dependency-checker
+source_code_url: https://github.com/johnjcsmith/bitrise-step-cocoapods-dependency-checker
+support_url: https://github.com/johnjcsmith/bitrise-step-cocoapods-dependency-checker/issues
+published_at: 2018-08-02T16:47:02.19911406+09:30
+source:
+  git: https://github.com/johnjcsmith/bitrise-step-cocoapods-dependency-checker.git
+  commit: f6a389a794df11e6923f335c4d219650e7060d41
+host_os_tags:
+- osx-10.10
+- ubuntu-16.04
```
Comment (BirmacherAkos) [inline steps/cocoapods-dependency-check/0.1/step.yml]: If this step is only for iOS than remove please the `- ubuntu-16.04` from `host_ost_tag`.

File: steps/cocoapods-dependency-check/0.1/step.yml [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+- dependency
+toolkit:
+  bash:
+    entry_file: step.sh
+deps:
+  brew:
+  - name: git
+  - name: wget
+  - name: dependency-check
+  apt_get:
```
Comment (BirmacherAkos) [inline steps/cocoapods-dependency-check/0.1/step.yml]: If this step is only for iOS than remove please the `apt_get:` from `deps`.

**Comment by BirmacherAkos** [comment]
Hi @johnjcsmith !
Thanks for your work.

Please use semantic versioning (MAJOR.MINOR.PATCH) for your Step. For example, 1.2.3.
You can read more about it here:
https://devcenter.bitrise.io/bitrise-cli/create-your-own-step/#naming-conventions-versioning-step-id-step-inputs-and-outputs

---

Please check my comments and do the requested changes.

---

Create a new version (with semantic versioning (MAJOR.MINOR.PATCH) ofc.) and share it again.

**Comment by johnjcsmith** [comment]
Closing now.

Will re-submit with changes.

Thanks.

**Pull request closed without merging**